content/events/2024-in-person-conference/_index.md

```diff
@@ -5,50 +5,86 @@ description = "Nordic RSE – Research Software Engineers in the Nordics – Con
 
 # Nordic-RSE in person conference 2024 <span style="color: gray;">#NordicRSEconf</span>
 
+
 **May 30-31**
 
 Registration will open in January 2024
 
 
+## What is it?
+
+This is the first Nordic RSE in person conference. We are an association of researchers who use software and people who write
+that software. So this is not like most academic conferences in that the focus is not on the results of research, but rather
+on the tools that underlie those results.
+
+Most research is already built on a foundation of sofware tools and the quality and reliability of those tools is key to
+producing reliable results. And the importance of software keeps growing. In this conference we want to talk about what
+pipelines you develop and use for your research, and how you use it, the software you develop, and how you got about
+developing it
+
+
+## Where
+
+On the [campus of Aalto University](https://maps.app.goo.gl/3TGvPnBw254T5X817) in Otaniemi, Espoo, Finland.
+
+
+### How you can help
+
+ - [Submit a contribution!](http://127.0.0.1:1111/events/2024-in-person-conference#call-for-contributions) The conference will be more interesting with your contribution on the agenda.
```

Comment:
Should this just '/events/2024-in-person-conference#call-for-contributions')?

Reply:
Whoops, thanks!